### PR TITLE
docs(tutorial): fixed code sample for toh-pt6

### PR DESCRIPTION
**Pull request by naffiq**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

The code snippet I've changed was titled as a service while it was a template

## What is the new behavior?

Correct header for the code snippet is now given

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by naffiq** [comment]
I signed it!

File: aio/content/tutorial/toh-pt6.md [hunk 1]
```diff
@@ -317,7 +317,7 @@ Each hero in the heroes list should have a delete button.
 Add the following button element to the `HeroesComponent` template, after the hero
 name in the repeated `<li>` element.
 
-<code-example path="toh-pt6/src/app/heroes/heroes.component.html" header="src/app/hero.service.ts" region="delete"></code-example>
+<code-example path="toh-pt6/src/app/heroes/heroes.component.html" header="src/app/heroes/heroes.component.ts" region="delete"></code-example>
```
Comment (kapunahelewong) [inline aio/content/tutorial/toh-pt6.md]: Could you change the file extension os `.ts` to `.html`?

**Review by kapunahelewong — CHANGES_REQUESTED** [review]
Good catch, @naffiq! Just one tiny comment on the change itself.

For the commit message, you can just make it:
`docs: fix code sample for toh-pt6`
You don't need the `tutorial` part and usually we put the verb in the present tense. 
(EDIT: the `tutorial` part is what's making CircleCI fail.)

Would you tag me when you've made the updates so I'll help you move it through the process? Thanks!

**Comment by kapunahelewong** [comment]
Edit: @naffiq I just realized that this is a duplicate of https://github.com/angular/angular/pull/31811.

**Pull request closed without merging**